service/awsconfig/v8/templates/cloudformation/guest/lifecycle_hooks.go

```diff
@@ -7,7 +7,7 @@ const LifecycleHooks = `{{define "lifecycle_hooks"}}
       AutoScalingGroupName:
         Ref: {{ .LifecycleHooks.Worker.ASG.Ref }}
       DefaultResult: CONTINUE
-      HeartbeatTimeout: 300
+      HeartbeatTimeout: 3600
```

Comment:
Changing from five minutes to one hour. This is to be more defensive for now. We are actually able to keep the instances for one or two days before AWS kills them completely. My reasoning here is that the interplay between the node-operator and the provider operators is asynchronous and in theory some time can pass by before we are able to fulfil the whole dance. We should definitely aim to make draining as fast as possible but for now we are not yet in a performance optinization stage and I want to get it reliably working first. We can adapt this here again later with ease as soon as we gathered more experiences and are confident about the overall process working out smoothly.